content/en/data_jobs/databricks.md

```diff
@@ -44,11 +44,21 @@ Follow these steps to enable Data Jobs Monitoring for Databricks.
 1. On the **Configure** tab, click **Add Databricks Workspace**.
 1. Enter a workspace name, your Databricks workspace URL, account ID, and the client ID and secret you generated.
    {{< img src="data_jobs/databricks/configure-workspace-form-m2m.png" alt="In the Datadog-Databricks integration tile, a Databricks workspace is displayed. This workspace has a name, URL, account ID, client ID, and client secret." style="width:100%;" >}}
+1. For getting visibility on your Databricks costs in Data Jobs Monitoring, provide a System Tables SQL Warehouse ID.
+   - The warehouse must have access to system tables (`system.billing.usage` for example).
```

Comment:
Can we just list the specific commands needed to be run to grant the permissions to all system tables so the customer doesn't need to figure it out?